agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -175,6 +175,8 @@
         // ignoreRemoteParentNotSampled is currently needed
         // because .NET SDK always propagates trace flags "00" (not sampled)
         public boolean ignoreRemoteParentNotSampled = true;
+        // TODO consider turning this on by default in 3.1.0
+        public boolean httpMethodInOperationName;
```

Comment:
If i remembered correctly, we used to have http method in operation name, then it got removed.. now we add it back as a config? i wonder what was the reason to remove it at the first place.

Reply:
it used to be HTTP GET + operationName